arlo-client/src/components/Audit/index.tsx

```diff
@@ -82,55 +83,97 @@ const Audit: React.FC<{}> = () => {
     refresh()
   }, [refresh])
 
+  const progressSidebar = (
+    <Sidebar
+      title="Audit Progress"
+      menuItems={[
+        {
+          title: 'Jurisdictions',
+          active: true,
+          state: 'live',
+        },
+      ]}
+    />
+  )
+  const aaSetupSidebar = <Sidebar title="Audit Setup" menuItems={menuItems} />
+  const jaSetupSidebar = (
+    <Sidebar
+      title="Audit Setup"
+      menuItems={[
+        {
+          title: 'Upload Ballot Manifest',
+          active: true,
+          state: 'live',
+        },
+      ]}
+    />
+  )
+
+  if (isAuthenticated)
+    return (
+      <Wrapper>
+        <ResetButton
+          electionId={electionId}
+          disabled={!audit.contests.length || isLoading}
+          updateAudit={updateAudit}
+        />
+        {viewMatch === 'setup' && meta!.type === 'audit_admin' && (
+          <>
+            {aaSetupSidebar}
+            <Setup stage={stage} refresh={refresh} menuItems={menuItems} />
+          </>
+        )}
+        {viewMatch === 'setup' && meta!.type === 'jurisdiction_admin' && (
+          <>
+            {jaSetupSidebar}
+            <p>Ballot manifest upload</p>
+          </>
+        )}
+        {viewMatch === 'progress' && (
+          <>
+            {progressSidebar}
+            <p>Progress view</p>
+          </>
+        )}
+        {viewMatch !== 'setup' && viewMatch !== 'progress' && (
+          <p>Round management view</p>
+        )}
+      </Wrapper>
+    )
```

Comment:
I'm a little worried by having data loading and rendering for the single-jurisdiction flow and the multi-jurisdiction flow in the same component. It's a bit confusing to read. I wonder if it would be simpler to have one high level conditional at the top of the component hierarchy that checks the logged in user type and delegates to one of three components:
- if anonymous user, render the single-jurisdiction flow
- if audit admin, render the audit admin view of the multi-jurisdiction flow
- if jurisdiction admin, render the jurisdiction admin view of the multi-jurisdiction flow

Each of these components would have its own routing and data loading internally.

Reply:
@jonahkagan I hear you, but that would also involve some massive file restructuring and would also involve splitting up and moving around a lot of tests and mocks. It would be cleaner in the end, and I do think that refactor would be a good thing to do, but right now I think it's a time commitment we don't need.